sklearn/feature_selection/_sequential.py

```diff
@@ -145,8 +153,8 @@ def fit(self, X, y):
                      "number of features, or a float in (0, 1] "
                      "representing a percentage of features to "
                      f"select. Got {self.n_features_to_select}")
-        if self.n_features_to_select is None:
```

Comment:
Shouldn't we keep the `None` case for now for backwards compatibility?

Reply:
fix it on https://github.com/scikit-learn/scikit-learn/pull/20145/commits/c6c4b2eb466c77e8ba7ad0f8cdf658ea8ab77449.

Simply, convert 'auto' when  `self.n_features_to_select` is `None`.
Should I send a message? 
(e.g. self.n_features_to_select=None is deprecated)

Reply:
Fixed not to change self.n_features_to_select to 'auto' just by issuing a warning when `n_features_to_select` is 'warn' on https://github.com/scikit-learn/scikit-learn/pull/20145/commits/83bd9a064557e3a5fd995c720f1bce09560f4d58 because assertion was occured in https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/utils/estimator_checks.py#L2731